ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -61,8 +62,22 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
     }
   });
 
+  @action
+  clearActivationErrors() {
+    this.activationErrors = null;
+  }
+
   @action
   onModalError(errorMsg: string) {
-    this.activationError = errorMsg;
+    if (DEBUG) console.error(errorMsg); // eslint-disable-line no-console
+
+    const errors = [errorMsg];
+
+    if (this.args.isHvdManaged) {
+      errors.push(
+        'Secrets Sync is available for Plus tier clusters only. Please check the tier of your cluster to enable Secrets Sync.'
+      );
+    }
+    this.activationErrors = errors;
```

Comment:
this is subtle, but it ensures we display both the raw API error and a more HVD-specific error in case the reason for the failure is due to the cluster having an incorrect tier. the result is 2 error banners, like so:

![Screenshot 2024-05-22 at 5 14 34 PM](https://github.com/hashicorp/vault/assets/903288/e9a0a2a8-ac8e-4dd0-b119-23e9eb23af92)

i considered showing only the tier error, but i didn't feel good about obfuscating the true server error just in case it occurred for any other reason. i also considered trying to keep both error messages within the same error banner but that would require edits to the `<MessageError />` component which started to feel like too much scope creep for an item we'd like to get in ASAP.

open to other ideas!